server/src/main/java/org/opensearch/common/remote/AbstractRemoteWritableEntityManager.java

```diff
@@ -24,6 +29,14 @@ public abstract class AbstractRemoteWritableEntityManager implements RemoteWrita
      */
     protected final Map<String, RemoteWritableEntityStore> remoteWritableEntityStores = new HashMap<>();
 
+    /**
+     * RemoteClusterStateSettings to add jitter in the download calls.
+     * Intentionally set to null to make sure the inheriting class configure it correctly.
+     */
+    protected RemoteClusterStateSettings remoteClusterStateSettings = null;
```

Comment:
This doesn't ensure inheriting classes will override this value. Should we instead have an abstract method that inheriting classes will need to implement?